pages/index.js

```diff
@@ -1,5 +1,8 @@
-import shoppingItems from "@/assets/shopping-items.json";
+import { useState } from "react";
+import Form from "@/components/Form";
 import ShoppingItemList from "@/components/ShoppingItemList";
+import shoppingItemsData from "@/assets/shopping-items.json";
+import { uid } from "uid";
 import styled from "styled-components";
 
 export default function HomePage() {
```

Comment:
We need to make sure we destructure the props here. 
These are the ones coming from `_app.js`

```suggestion
export default function HomePage({ shoppingItems, onCreateItem}) {
```